packages/api/cms-api/schema.gql

```diff
@@ -228,6 +228,8 @@ type DamFile {
   license: DamFileLicense
   createdAt: DateTime!
   updatedAt: DateTime!
+  sourceId: String!
+  sourceType: String!
```

Comment:
should this be named importSourceType and importSourceId?

Reply:
thats a good idea